ftw/contentpage/tests/test_newslisting_rss.py

```diff
@@ -45,3 +46,11 @@ def test_newslisting_rss_items(self):
         self.assertIn(link,
                       self.browser.contents,
                       'Did not found the link tag for the news')
+
+    @browsing
+    def test_newsitem_contains_pubdate(self, browser):
+        browser.open(self.newsfolder, view='news_rss_listing')
+        self.assertEqual(
+            self.news.getEffectiveDate().strftime('%a, %e %b %Y %H:%M:%S %z'),
```

Comment:
There is no timezone (`%z`) in the test, therefore a trailing space here.
Since the `got` part of the assertion is stripped, the assertion fails.